packages/campaign/reducer.js

```diff
@@ -86,10 +86,16 @@ export default (state = initialState, action) => {
     case actionTypes.PUSHER_CAMPAIGN_UPDATED: {
       const { campaign } = state;
       const updatedCampaign = campaignParser(action.campaign);
+      const campaignToLoad =
+        updatedCampaign.id === campaign.id ? updatedCampaign : campaign;
+      const channels = campaign.channels || state.campaign.channels;
```

Comment:
aren't `campaign.channels` and `state.campaign.channels` the same thing? because on line 87 `const { campaign } = state;`

```suggestion
      const channels = campaign.channels;
```

Reply:
Oh! great catch @karelm 🤦‍♀️ what I meant was actually:

`const channels = updatedCampaign.channels || campaign.channels;`